### PR TITLE
Update: the intro to storybook vue/chinese documentation

### PR DESCRIPTION
**Pull request by fengnzl**
It will be my first time contributing to a project so I hope I can be useful! I updated vue/chinese get-started and simple-component documentations.

### CODE REVIEW

**Comment by jonniebigodes** [comment]
@fengnzl, this is fantastic. Thank you so much for the time and effort you've put into this pull request and for helping us update the translation 🙏 ! With that, I wanted to ask if you're also okay with updating the remaining tutorial sections. As you already started the work in this branch, feel free to use it to make the rest of the changes. And also, once it's done, ping me so that I can check with our community for a proofreader to take a look at this and get it merged and published. Does that work for you?

Looking forward to hearing from you.

Hope you have a great day.

Stay safe

**Comment by fengnzl** [comment]
@jonniebigodes No problem.I will translate the rest of the intro of Vue part-time, and ping you as soon as it's finished.

**Comment by jonniebigodes** [comment]
@fengnzl that's great to hear, I've been checking on and off the progress, and it's looking good so far. I'll take a look once you've finished, and we'll go from there.

**Comment by fengnzl** [comment]
@jonniebigodes I've finished translating the intro of Vue. Please take a look and check this.  But It seems something went wrong in CI, I checked the error message and it maybe not caused by me. If there is something that needs to be fixed, I would handle that.

**Comment by jonniebigodes** [comment]
@fengnzl, thanks for taking care of the translation in such a short amount of time 🙏 ! I'm going to take a pass at it shortly. If you don't mind taking care of it, one thing left to update is to change the value [here](https://github.com/chromaui/learnstorybook.com/blob/master/gatsby-config.js#L42) from `6.3` to `6.5`. So that once the translation is merged, it doesn't incorrectly mention that is outdated. 

Regarding the CI workflow, don't worry about it. I'm checking it to see what may be happening on our end. Thanks for letting me know, nonetheless.

**Comment by fengnzl** [comment]
The version has been updated.

**Comment by fengnzl** [comment]
@jonniebigodes In case you don't see the information above. I @ you again. and hope to hear the message that can be merged.

**Comment by jonniebigodes** [comment]
@fengnzl I saw the update. Sorry for not following up with you on this. I will reach out to the community later today and see if we can find a proofreader. Tomorrow morning I'll take a look at the translation in more detail and follow up with you if any changes are required on my end. Sounds good?

**Comment by fengnzl** [comment]
@jonniebigodes That's ok. I'm glad to modify the required changes.

**Review by jonniebigodes — CHANGES_REQUESTED** [review]
@fengnzl from me, just one small item and this should be good to go. I sent out a callout to our community to see if someone was interested in proofreading it. To avoid having this pr being blocked from merging, I'm going to put a timer on it, so to say, if no one comes forward and follows up with you and me and proofreads it by end of next week. I'll merge it, and we'll polish it up with follow-up pull requests. Does that work for you?

Looking forward to hearing from you. 

Hope you have a great week.

Stay safe

File: content/intro-to-storybook/vue/zh-CN/deploy.md
Comment (fengnzl) [reply content/intro-to-storybook/vue/zh-CN/deploy.md:105]: Ok, I will fix it later and check other links.

**Comment by fengnzl** [comment]
@jonniebigodes Hello, I wonder when it will be merged, or is there anything to be fixed?

**Comment by jonniebigodes** [comment]
@fengnzl , I haven't forgotten about you, don't worry about it. I'll take a last pass at it today, merge it if nothing is required and follow up with you on it, as no one came forward to proofread it. Don't want to get you blocked on this as you already have it pretty much wrapped up.

**Review by jonniebigodes — APPROVED** [review]

**Comment by jonniebigodes** [comment]
@fengnzl I just took a final pass at your incredible work, and it looks good on my end. A couple of things I would like your feedback on. First, can you jump into our [Discord Server](https://discord.gg/storybook) and message me (same username) for a small item related to this? Second, I'm going to update the call to translations [issue](https://github.com/chromaui/learnstorybook.com/issues/3) we have and reference you as a translator/contributor so that when future pull requests addressing your native language surface, we can sync up with you and help us out in proofreading it. Does that work for you?

Looking forward to hearing from you. I hope you have a fantastic day, and extremely thankful for this fantastic contribution.

Stay safe

**Comment by jonniebigodes** [comment]
@fengnzl, thanks for following up with me, and that's great to hear. We appreciate your willingness to help us proofread any follow-up pull requests. And don't worry about the Discord item for now. We have an alternative. I will reach out to you via the email you have listed in your GitHub profile, and we'll go from there. Sounds good? Be on the lookout for it.

I hope you have a great day.

Stay safe

**Pull request merged**
